flight/targets/colibri/fw/chconf.h

```diff
@@ -405,7 +405,7 @@
  *          @p panic_msg variable set to @p NULL.
  */
 #if !defined(CH_DBG_ENABLE_STACK_CHECK) || defined(__DOXYGEN__)
-#define CH_DBG_ENABLE_STACK_CHECK       TRUE
```

Comment:
Note the dynamic analysis that's been the practice here is really flawed-- we've found tasks that use 400 bytes more than you could usually get into in testing through infrequently taken paths.

The static stack analysis really helps, but the tool that I've worked on at dR requires pretty specialized knowledge and a lot of time to use, and isn't perfect (usually can't follow function pointers and doesn't have knowledge of what the system libraries use).

If you're going to undertake a static stack survey... or greatly increase stacks (a few hundred bytes each), that would address one of the primary concerns with merging this.